packages/components/src/date-time/time/styles.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+/**
+ * Internal dependencies
+ */
+import { COLORS, CONFIG } from '../../utils';
+import { space } from '../../ui/utils/space';
+import {
+	Input,
+	BackdropUI,
+} from '../../input-control/styles/input-control-styles';
+import NumberControl from '../../number-control';
+import SelectControl from '../../select-control';
+
+export const Wrapper = styled.div``;
+
+export const Fieldset = styled.fieldset`
+	border: 0;
+	margin: 0 0 ${ space( 2 * 2 ) } 0;
+	padding: 0;
+
+	&:last-child {
+		margin-bottom: 0;
+	}
+`;
+
+export const Legend = styled.legend`
+	margin-bottom: ${ space( 2 ) };
+	padding: 0;
+`;
+
+export const TimeWrapper = styled.div`
+	direction: ltr;
+	display: flex;
+`;
+
+const baseInput = css`
+	&&& ${ Input } {
+		padding-left: ${ space( 2 ) };
+		padding-right: ${ space( 2 ) };
+		text-align: center;
+	}
+`;
+
+export const HoursInput = styled( NumberControl )`
+	${ baseInput }
+
+	width: 35px;
+
+	&&& ${ Input } {
+		padding-right: 0;
+	}
+
+	&&& ${ BackdropUI } {
+		border-right: 0;
+		border-top-right-radius: 0;
+		border-bottom-right-radius: 0;
+	}
+`;
+
+export const TimeSeparator = styled.span`
+	border-top: ${ CONFIG.borderWidth } solid ${ COLORS.gray[ 700 ] };
+	border-bottom: ${ CONFIG.borderWidth } solid ${ COLORS.gray[ 700 ] };
+	line-height: calc(
+		${ CONFIG.controlHeight } - ${ CONFIG.borderWidth } * 2
+	);
+	display: inline-block;
+`;
+
+export const MinutesInput = styled( NumberControl )`
+	${ baseInput }
+
+	width: 35px;
+
+	&&& ${ Input } {
+		padding-left: 0;
+	}
+
+	&&& ${ BackdropUI } {
+		border-left: 0;
+		border-top-left-radius: 0;
+		border-bottom-left-radius: 0;
+	}
+`;
+
+// Ideally we wouldn't need a wrapper, but can't otherwise target the
+// <BaseControl> in <SelectControl>
+export const MonthSelectWrapper = styled.div`
+	flex-grow: 1;
+`;
+
+export const MonthSelect = styled( SelectControl )`
+	height: 36px;
+
+	// Ideally would target the component instead of a classname.
+	.components-select-control__input {
+		line-height: 30px;
+	}
```

Comment:
Have you tried using the `Select` styled component from `SelectControl` as the selector here? (excuses for the "selection" of terms used).

Something like

```diff
diff --git a/packages/components/src/date-time/time/styles.ts b/packages/components/src/date-time/time/styles.ts
index 7f8ff90de4..567f32c0c5 100644
--- a/packages/components/src/date-time/time/styles.ts
+++ b/packages/components/src/date-time/time/styles.ts
@@ -15,6 +15,7 @@ import {
 } from '../../input-control/styles/input-control-styles';
 import NumberControl from '../../number-control';
 import SelectControl from '../../select-control';
+import { Select } from '../../select-control/styles/select-control-styles';
 
 export const Wrapper = styled.div``;
 
@@ -96,8 +97,7 @@ export const MonthSelectWrapper = styled.div`
 export const MonthSelect = styled( SelectControl )`
 	height: 36px;
 
-	// Ideally would target the component instead of a classname.
-	.components-select-control__input {
+	${ Select } {
 		line-height: 30px;
 	}
 `;
```

(Haven't tested on my local machine though)